### PR TITLE
Feature/adri/update api

### PR DESCRIPTION
**Pull request by adriaulia**
add function api in node js sdk

### CODE REVIEW

File: handlers/bnpl.js [hunk 1]
```diff
@@ -1,14 +1,66 @@
...
+
+  const response = await httpRequest(options);
+  return response;
+};
+
+const requestLimitOTP = async (params) => {
+  const checkAuth = validateAuth(params);
+  if (checkAuth) return responseJSON(checkAuth);
+
+  console.log(params)
```
Comment (rilexkoinworks) [inline handlers/bnpl.js:42]: please remove every console.log

File: handlers/bnpl.js [hunk 2]
```diff
@@ -1,14 +1,66 @@
...
   return response;
 };
 
-module.exports = { requestLimit };
\ No newline at end of file
+const generateToken = async (params) => {
+  
+  const { data, headers } = params;
+
+  data["type"] = "user";
```
Comment (rilexkoinworks) [inline handlers/bnpl.js:26]: change `data["type"]` & `data["scopes"]` into `{ type, scopes } = data`;

**Review by rilexkoinworks — COMMENTED** [review]
please check my comments

File: handlers/bnpl.js (hunk 1 above)
Comment (adriaulia) [reply handlers/bnpl.js:42]: wahh sorry mas ampun 🙏🏻

File: handlers/bnpl.js (hunk 2 above)
Comment (adriaulia) [reply handlers/bnpl.js:26]: mas @rilexkoinworks , ini buat set data mas, bukan nampung data

kalau di buat seperti ini gmn mas?

`data.type = "value"`
`data.scopes = "value"`

Comment (rilexkoinworks) [reply handlers/bnpl.js:26]:
```
{ type, scopes } = data;
type = "value";
scopes = "value";
```

**Comment by adriaulia** [comment]
has been commit and push ya mas🙏🏻

**Review by rilexkoinworks — APPROVED** [review]

**Pull request merged**